v3/state.py

```diff
@@ -52,14 +53,16 @@ def __init__(
         self.data_path = str(Path(f"{PACKAGEDIR}/data").resolve())
         checkPath("", self.data_path)
 
-        # we strip the "uniswap_v3_factory"
-        # and "uniswap_v3_pool" bc its unneeded
-        self.tables = [
-            "factory_pool_created",
-            "pool_initialize_events",
-            "pool_swap_events",
-            "pool_mint_burn_events",
-        ]
+        # tables to update
+        if type(tables) != list:
```

Comment:
why are you checking the type here?

Reply:
are you just checking if tables is None?

Reply:
checking if None but also that it is the expected type (a list)

didn't want to have any breaking changes either. probably a better way to do this